worldeditadditions_core/register/check.lua

```diff
@@ -0,0 +1,14 @@
+function worldeditadditions_core.register_command(def)
+	local def = table.copy(def)
+	assert(name and #name > 0)
+	assert(def.privs)
+	def.require_pos = def.require_pos or 0
+	assert(def.require_pos >= 0 and def.require_pos < 3)
+	if def.params == "" and not def.parse then
+		def.parse = function(param) return true end
```

Comment:
Teensy suggestion 'cause I'm picky
```suggestion
		def.parse = function(params_text) return true end
```

Reply:
Open a new issue "wea.core devlopment" and suggest this there 😃.

Reply:
Yeah, probably - but would be so much easier to commit this here :P